app/src/main/java/com/mapbox/vision/examples/activity/ar/ArNavigationActivity.kt

```diff
@@ -216,6 +219,26 @@ class ArNavigationActivity : AppCompatActivity(), RouteListener, ProgressChangeL
         return routePoints.toTypedArray()
     }
 
+    private fun String?.mapToManeuverType(): ManeuverType = when(this) {
```

Comment:
Let's unify it as a single extension. Also, I'd make it extension of `StepManeuver` instead of String to avoid cluttering in code completion for strings

Reply:
Done